### DISCUSSION TITLE
Cutomise Board Item style

### DISCUSSION THREAD

**Discussion by rajpchaudhary** [cloudscape-design/components#1931] [Q&A] [answered]
Hi There,
I want to customize the style of the Board Item look and feel. e.x "want to remove Border radius ".
is there any way to manage and add customized CSS for Board components?

**Comment by abdhalees** [comment]
you can change the containers border radius with themeing https://cloudscape.design/foundation/visual-foundation/theming/

change the value of `$border-radius-container` but be aware that it will be changed for all container like components like: `Container` and `Cards`

**Reply by just-boris** [reply]
We do not offer theming features on board components and I am not sure we want to start

**Comment by just-boris** [comment, marked as answer]
Discussion moved into the issue: https://github.com/cloudscape-design/components/issues/1935#issuecomment-1938528070
